Rocket.Chat/de.lproj/Localizable.strings

```diff
@@ -134,11 +154,8 @@
 "chat.messages.pinned.list.title" = "Angeheftete Nachrichten";
 "chat.messages.starred.list.title" = "Favorisierte Nachrichten";
 
-// New Room List
-"new_room.cell.public_channel.title" = "Öffentlicher Kanal";
-"new_room.cell.public_channel.description" = "Jeder Benutzer kann einem öffentlichen Raum beitreten.";
-"new_room.cell.read_only.title" = "Schreibgeschützter Kanal";
-"new_room.cell.read_only.description" = "Nur Moderatoren oder Admins können in schreibgeschützten Kanälen schreiben";
-"new_room.group.channel.name" = "Channel Name";
-"new_room.cell.channel_name.title" = "Channel Name";
-"new_room.group.channel.footer" = "Der Name muss komplett in Kleinbuchstaben und kürzer als 22 Zeichen sein.";
+// Settings
+"myaccount.settings.title" = "Settings";
+"myaccount.settings.contactus" = "Contact us";
+"myaccount.settings.license" = "License";
+"myaccount.settings.version" = "Version: %@ (%@)";
```

Comment:
@TheReal1604 I know it sucks, but could you help us translating this new strings? We're working to integrate with LingoHub soon.

Reply:
Sure! I take a look...